src/Migrations/Version20181204110357.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20181204110357 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->createTableAddress($schema);
+        $this->createTableContact($schema);
+        $this->createTableContactAddress($schema);
+        $this->modifyTableOrganisation($schema);
+    }
+
+    private function createTableAddress(Schema $schema): void
+    {
+        $schema->createSequence('address_id_seq');
+
+        $table = $schema->createTable('address');
+        $table->addColumn('id', Type::GUID);
+        $table->addColumn('street_address', Type::STRING);
```

Comment:
I think we should specify the max length even if we use the default value (just to be sure)